edit recipe and delete recipe

AsianHam opened the pull request:
added an edit recipe page that fills in initial values to the add recipe page

created a functioning delete button with confirmation prompt

AsianHam committed:
editRecipe and delete recipe

added an edit recipe page that fills in initial values to the add recipe page

created a functioning delete button with confirmation prompt

Jacksondr5 commented on src/App.tsx:
```diff
@@ -53,14 +54,17 @@ function App() {
   const { Panel } = Collapse;
 
-  const genExtra = () => (
+  const genExtra = (recipeId: number) => (
```
This name isnt descriptive

Jacksondr5 commented on src/App.tsx:
```diff
@@ -74,14 +78,19 @@ function App() {
 
+  const deleteRecipe = (recipeId: number) => {
+    DeleteRecipe(recipeId);
+    window.location.reload();
```
Replace this with a call to getAllRecipes

AsianHam committed:
new page and edit page split

separated edit page and new page into two different files

separated the form into a separate file
